Week-1/assignment2/style.css

```diff
@@ -0,0 +1,158 @@
+* {
+  font-family: "Lato", sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  /* height: 100%;

```

Comment:
try to remove the commented code everywhere unless its some information you want to pass on